types/leaflet-offline/leaflet-offline-tests.ts

```diff
@@ -0,0 +1,6 @@
+import * as L from 'leaflet';
+import 'leaflet-offline';
+const map: L.Map = L.map('map-container');
+L.tileLayer('http://{s}.tile.osm.org/{z}/{x}/{y}.png', {
```

Comment:
Is this testing any of the new declarations?  I might be missing something, but it looks like it would work the same without `import 'leaflet-offline';`.

Reply:
If it works without importing 'leaflet-offline'